editor/src/clj/editor/code/data.clj

```diff
@@ -2520,12 +2520,19 @@
            :type :execution-marker
            :location-type location-type)))
 
-(defn breakpoint [lines ^long row]
+(defn make-breakpoint-region [lines ^long row]
   (let [line-start (text-start lines row)
         line-end (count (lines row))]
     (assoc (->CursorRange (->Cursor row line-start)
                           (->Cursor row line-end))
-      :type :breakpoint)))
+      :type :breakpoint
+      :enabled true)))
+
+(defn get-breakpoint-region [lines regions ^long row]
+  (some #(when (and (= (:type %) :breakpoint)
+                    (= (get-in % [:from :row]) row))
```

Comment:
Nitpick: we typically avoid `get-in` in favor of multiple lookups like `(:row (:from %))` for better performance.